### ISSUE TITLE
options are blank after disabling and enabling in add-ons on fennec

### ISSUE THREAD

**Issue by kr1shnas** [zalmoxisus/redux-devtools-extension#380]
device: lyf flame 3
os: android 5.1
version: 2.15.1

it does not show in toolbar.
there are no options in add-ons.
after tapping disable and enable.
it shows "options" in bold font and no another text.

repro:

fennec 54.0.1 bn-in
fennec 55.0.b5 bn-in
fennec 56.0a1 [ build: 20170703150223 ] bn-in

**Comment by zalmoxisus** [comment]
Not sure I understand the issue. Are you using Firefox extension on android? I didn't try that.

**Comment by kr1shnas** [comment]
i am using on android.
there are no options in addons.
options in bold is visible after disable and enable addon.

anyhow, i have come across your addon on bugzilla, perhaps reporter assumed this as a webextension.

is this just addon or webextension.
https://wiki.mozilla.org/WebExtensions

**Comment by zalmoxisus** [comment]
Yes, it's WebExtension. I didn't try it on Android but if it doesn't work properly, will allow it only from pc from the next version. I guess there's no DevTools panel either, so we cannot show it.

**Comment by kr1shnas** [comment]
as far as i know, webextensions are os independent.

herewith, i am providing link of logcat using logview addon.
[logs](https://gist.github.com/kr1shnas/fdab1d37feb40371331663df6d42b6b0)

perhaps it might help.
